### DISCUSSION TITLE
Updated Version Compatibility

### DISCUSSION THREAD

**Discussion by postal302** [roleoroleo/yi-hack-Allwinner#399] [Q&A] [answered]
First let me say this is an awesome utility, and I'm incredibly impressed with the work you've done and all the support you continue to provide!
It appears the newer Outdoor 1080p cameras have updated hardware, or at least updated boards. I've seen some conversations here that discuss the QG2101A chip, but I wasn't able to determine if the outcome is they're compatible or not.
![0515231551](https://github.com/roleoroleo/yi-hack-Allwinner/assets/43799629/996630e2-dd21-46d1-81fb-9e8b602d60f9)
![0515231541](https://github.com/roleoroleo/yi-hack-Allwinner/assets/43799629/5e586737-72eb-464e-a798-f73253de045c)
I haven't plugged this one in or tried to hack it yet. And I'm new to these so I'm stepping cautiously.
The board says HG31A_MB_V1.2 and the chip is a QG2101A.
Is it known if this version is compatible?
Thanks for the insight, and for making an amazing tool!

**Comment by roleoroleo** [comment]
Please, try to make a backup of the flash and share it with me.
I will check the exact model.

**Comment by postal302** [comment]
...spoke too soon. Just dumped the backup. Will upload in a bit.

**Comment by postal302** [comment]
Created a repository with the dumped firmware.
https://github.com/postal302/Yi-H31GA-Backup
Thank you for your continuous hard work on these!

**Reply by roleoroleo** [reply, marked as answer]
The cam is a h30ga with fw 9.0.26.
Already supported.

**Reply by postal302** [reply]
Awesome! @roleoroleo you're a hero!
